addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16016,7 +16016,19 @@ msgctxt "#35010"
 msgid "Peripheral libraries"
 msgstr ""
 
-#empty strings from id 35011 to 35048
+#. Title of dialog to ask users if they would like to configure a new controller
+#: xbmc/input/joysticks/dialogs/GUIDialogNewJoystick.cpp
+msgctxt "#35011"
+msgid "New controller detected"
+msgstr ""
+
+#. Text of dialog to ask users if they would like to configure a new controller
+#: xbmc/input/joysticks/dialogs/GUIDialogNewJoystick.cpp
+msgctxt "#35012"
+msgid "Would you like to configure your controller? This can be done later in System Settings."
```

Comment:
I'm not a native speaker, but for me the following wording would be more clear. Also, IMO the question the user has to base the decision on should ideally be the last part of the message in a yes/no dialog so that the button labels are the immediate answer to the question.

`A new controller has been detected and can be configured. Configuration can be done at any time by navigating to "Settings -> System Settings -> Input". Would you like to configure it now?` [YES] [NO]

Likely needs improvements as I'm still half asleep, but you get the idea.